doc/landing-page/index.html

```diff
@@ -0,0 +1,259 @@
+                                    <span data-feather="book" width="28" height="28"></span>
+                                </div>
+                            </div>
+                            <div class="media-body">
+                                Helpful boilerplates allowing developers to focus on use-cases rather than code organization
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-sm-6">
+                        <div class="media">
+                            <div class="icon-box">
+                                <div class="icon-box-inner">
+                                    <span data-feather="git-pull-request" width="28" height="28"></span>
+                                </div>
+                            </div>
+                            <div class="media-body">
+                                REPL - to quickly and easily interact with ASA and Smart Contracts
+                            </div>
+                        </div>
+                    </div><div class="col-sm-6">
+                        <div class="media">
+                            <div class="icon-box">
+                                <div class="icon-box-inner">
+                                    <span data-feather="zap" width="28" height="28"></span>
+                                </div>
+                            </div>
+                            <div class="media-body">
+                                Runtime - to test smart contracts very fast without being on the real blockchain
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<!--Features Section-->
+<div class="container">
+    <div class="row">
+        <div class="col-md-9 mx-auto">
+            <h2 class="text-center text-md-left">We are waiting on your feedback, contact us using algo-builder
+                <a href="https://github.com/scale-it/algo-builder/issues">
+                    <span class="text-secondary">GitHub Issues</span></a>
+            </h2>
+        </div>
+    </div>
+</div>
+
+<!--footer-->
+<footer class="py-5 bg-light">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 text-center">
+                <ul class="list-inline">
+                    <li class="list-inline-item"><a href="https://github.com/scale-it/algo-builder/issues">Contact Us</a></li>
+                    <li class="list-inline-item"><a href="../algob/index.html">API Docs</a></li>
+                    <li class="list-inline-item"><a href="https://github.com/scale-it/algo-builder">Fork algo-builder on GitHub</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="row my-2">
+            <div class="col-md-4 mx-auto text-muted text-center small-xl">
+                &copy; 2020 Algo Builder - All Rights Reserved
```

Comment:
instead of 2020, let's use JS to set a right year here.

Reply:
Yes